zqd unit tests: 409 another write in progress

I've observed this on zq master failure. The zqd handler test fails with the following message:

```
--- FAIL: TestSpaceInfo (0.00s)
112 handlers_test.go:254: 
113 Error Trace:	handlers_test.go:254
114 Error:      	Received unexpected error:
115 status code 409: another write is already in progress
116 Test:       	TestSpaceInfo
117
FAIL
```

https://github.com/brimsec/zq/runs/1428003522?check_suite_focus=true

I think I have seen this before:

--- a/ppl/zqd/storage/filestore/filestore.go
+++ b/ppl/zqd/storage/filestore/filestore.go
@@ -198,10 +198,10 @@ type info struct {
 func (s *Storage) readInfoFile() error {
 	var inf info
 	if err := fs.UnmarshalJSONFile(s.join(infoFile), &inf); err != nil {
-		if os.IsNotExist(err) {
-			return nil
+		if !os.IsNotExist(err) {
+			return err
 		}
-		return err
+		inf.AlphaMigrated = true
 	}
 	s.span = nano.NewSpanTs(inf.MinTime, inf.MaxTime)
 	s.alphaMigrated = inf.AlphaMigrated

